code/_onclick/hud/hud_datum.dm

```diff
@@ -204,6 +213,9 @@
 	return
 
 /mob/proc/hide_hud()
+	if(HAS_TRAIT(src, TRAIT_KNOCKEDOUT))
```

Comment:
this should probably check that the mob is living and not an admin

Reply:
Modified to check living. Admins are not immune.